README.md

```diff
@@ -59,15 +95,55 @@ This scripts excludes from the mirror:
 - source RPMs
 - development RPMs
 - debugging-symbols RPMs
+- a few large RPMs only useful as build dependencies: ocaml, golang,
+  java
```

Comment:
```suggestion
  java, xs-opam-repo, etc.
```

Reply:
I considered `xs-opam-repo` to be part of `ocaml`

Reply:
ok